index.html

```diff
@@ -1,64 +1,235 @@
-  <li><a href="./samples/audio/shiny-drum-machine.html">shiny-drum-machine.html</a></li>
-  <li><a href="./samples/audio/simple-mixing.html">simple-mixing.html</a></li>
-  <li><a href="./samples/audio/simple.html">simple.html</a></li>
-  <li><a href="./samples/audio/stereopanner.html">stereopanner.html/</a></li>
-  <li><a href="./samples/audio/subtractive.html">subtractive.html</a></li>
-  <li><a href="./samples/audio/synthetic-clock.html">synthetic-clock.html</a></li>
-  <li><a href="./samples/audio/test-audio-tag.html">test-audio-tag.html</a></li>
-  <li><a href="./samples/audio/test-tones.html">test-tones.html</a></li>
-  <li><a href="./samples/audio/test-waveshaper.html">test-waveshaper.html</a></li>
-  <li><a href="./samples/audio/timeline.html">timeline.html</a></li>
-  <li><a href="./samples/audio/tone-editor.html">tone-editor.html</a></li>
-  <li><a href="./samples/audio/tone.html">tone.html</a></li>
-  <li><a href="./samples/audio/visualizer-chorus.html">visualizer-chorus.html</a></li>
-  <li><a href="./samples/audio/visualizer-gl/">visualizer-gl/</a></li>
-  <li><a href="./samples/audio/visualizer-gl.html">visualizer-gl.html</a></li>
-  <li><a href="./samples/audio/visualizer-live-doubling.html">visualizer-live-doubling.html</a></li>
-  <li><a href="./samples/audio/visualizer-live.html">visualizer-live.html</a></li>
-  <li><a href="./samples/audio/visualizer-noise.html">visualizer-noise.html</a></li>
-  <li><a href="./samples/audio/visualizer2-gl.html">visualizer2-gl.html</a></li>
-  <li><a href="./samples/audio/waveshaper-oversample.html">waveshaper-oversample.html</a></li>
-  <li><a href="./samples/audio/wavetable-synth.html">wavetable-synth.html</a></li>
-  <li><a href="./samples/audio/wavetable-synth2.html">wavetable-synth2.html</a></li>
-  <li><a href="./samples/audio/zero-crossings.html">zero-crossings.html</a></li>
-</ul>
-</body></html>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+  <!--
+  Copyright (c) 2017 The Chromium Authors. All rights reserved.
+  Use of this source code is governed by a BSD-style license that can be
+  found in the LICENSE file.
+  -->
+  <head>
+    <title>
+      Chrome WebAudio Samples
+    </title>
+    <link rel="stylesheet" href="resources/base.css">
+  </head>
+  <body>
+    <h1>
+      Chrome WebAudio Samples
+    </h1>
+    <h2>
+      Quick links
+    </h2>
+    <ul>
+      <li>
+        <a href="https://webaudio.github.io/web-audio-api/">Web Audio API spec
+        </a>
+      </li>
+      <li>
+        <a href="https://github.com/GoogleChrome/web-audio-samples">Chrome
+        WebAudio Sample GitHub repo</a>
+      </li>
+      <li>
+        <a href="samples/audio/index.html">Sample index page (circa 2010)</a>
+      </li>
+      <li>
+        <a href="samples/audio/samples.html">Third-party samples</a>
+      </li>
+      <li>
+        <a href="stress-test/boxes">Box2D Stress Test</a>
+      </li>
+      <li>
+        <a href="https://hoch.github.io/canopy/">Canopy</a>
```

Comment:
I think canopy should be right after the spec link.

Possibly separate out the spec link and canopy from the other items into it's own list. Kind of like:

- spec
- canopy
- github 

[blank line]
- old index
- 3rd party
- stress test